docs/quickstart.md

```diff
@@ -4,11 +4,14 @@ This guide assumes no prior knowledge of pytorch or any other deep learning fram
 It is intended to be a very quick overview of the high level API that tinygrad provides.
 
 This guide is also structured as a tutorial which at the end of it you will have a working model that can classify handwritten digits.
+The code for this tutorial is also available in [quickstart.py](/examples/quickstart.py).
```

Comment:
Why is it in two places? They should be merged.

Reply:
merging